cdap-ui/app/features/streams/controllers/tabs/status-ctrl.js

```diff
@@ -1,7 +1,8 @@
 angular.module(PKG.name + '.feature.streams')
-  .controller('StreamsDetailStatusController', function($scope, $state, myHelpers, MyDataSource, myStreamApi) {
+  .controller('StreamDetailStatusController', function($scope, $state, myHelpers, MyDataSource, myStreamApi) {
     var dataSrc = new MyDataSource($scope);
-
+    this.storage = null;
+    this.events = null;
```

Comment:
The default value is not used anywhere. In the template we check for null value and use a &mdash; to indicate that we don't have any value for it. Thought null would make sense since if we don't get any metrics for it its default is null (which means we don't have any metric for it). Not sure if backend would return a value of 0 in which case we should display 0 instead of &mdash;